backend/accounts/transaction_mock.go

```diff
@@ -0,0 +1,395 @@
+// Code generated by moq; DO NOT EDIT.
```

Comment:
@x1ddos i committed this by accident :( it's not used anywhere. will delete soon

Reply:
ah, wasn't sure.
sounds good! no worries. :+1: